rostwitter get_access_token.py not found

I'm re-configuring PR2 and I must re-issue access token for twitter.
when I execute `rosrun rostwitter tweet.py`, an error occured.

```
[ERROR] [WallTime: 1494841351.862268] "/var/lib/robot/twitter_account_pr2jsk.yaml" not found
[ERROR] [WallTime: 1494841351.863620] $ rosrun rostwitter get_access_token.py
[ERROR] [WallTime: 1494841351.864082] cat /var/lib/robot//var/lib/robot/twitter_account_pr2jsk.yaml <<EOF
[ERROR] [WallTime: 1494841351.864569] CKEY: xxx
[ERROR] [WallTime: 1494841351.865008] CSECRET: xxx
[ERROR] [WallTime: 1494841351.865596] AKEY: xxx
[ERROR] [WallTime: 1494841351.865991] ASECRET:
```
but I coud not find `get_access_token.py`. How can we use this?
CC: @k-okada 
https://github.com/jsk-ros-pkg/jsk_3rdparty/blob/master/rostwitter/scripts/tweet.py#L129

--- a/rostwitter/scripts/tweet.py
+++ b/rostwitter/scripts/tweet.py
@@ -126,7 +126,7 @@ def load_oauth_settings():
         ASECRET = key['ASECRET']
     except IOError as e:
         rospy.logerr('"%s" not found'%account_info)
-        rospy.logerr("$ rosrun rostwitter get_access_token.py")
+        rospy.logerr("$ get access token from https://apps.twitter.com/")
         rospy.logerr("cat /var/lib/robot/%s <<EOF"%account_info)
         rospy.logerr("CKEY: xxx")
         rospy.logerr("CSECRET: xxx")

